ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,59 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, Algorithm
+using ITensorNetworks:
+  ITensorNetworks,
+  ITensorNetwork,
+  vertextype,
+  vertex_data,
+  externalinds,
+  contraction_sequence
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function prepare_einexpr(tn::ITensorNetwork)
+  IndexType = Any
+  VertexType = vertextype(tn)
+
+  tensors = EinExpr{IndexType}[]
+  tensor_map = Dict{Set{IndexType},VertexType}()
```

Comment:
Could we come up with a better name for this mapping? It required me to read to type signature to understand what it might represent. Maybe `tensor_inds_to_vertex` or `inds_to_vertex` (I guess it is identifying which vertex a tensor is on based on the indices?).